instrumentation/httpd/src/otel/opentelemetry.cpp

```diff
@@ -85,8 +85,17 @@ void initTracer()
     processor = std::make_shared<sdktrace::SimpleSpanProcessor>(std::move(exporter));
   }
 
+  opentelemetry::sdk::resource::ResourceAttributes resAttrs({{"service.name", "httpd"}});
```

Comment:
From the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#service):
```
[1]: MUST be the same for all instances of horizontally scaled services. If the value was not specified, SDKs MUST fallback to u nknown_service: concatenated with process.executable.name, e.g. unknown_service:bash. If process.executable.name is not available, the value MUST be set to unknown_service.
```

Reply:
Yes. Thanks. Removed.